backend/fleet_management/migrations/0011_add_default_groups.py

```diff
@@ -0,0 +1,32 @@
+
+from django.db import migrations, connections
+from django.contrib.auth.models import Group
+from ..models import User
+
+
+from ..constants import Groups
+
+
+def create_groups(apps, schema_editor):
+    with connections['default'].cursor() as cursor:
```

Comment:
This line is not needed after my suggestion, can you remove it and indent the lines below to the left? Thanks!

Reply:
👍